cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslHelp.groovy

```diff
@@ -0,0 +1,33 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+st.contentType(value: "text/html;charset=UTF-8")
+
+l.layout(title:_("Jenkins Workflow Reference"),permission:app.READ) {
+
+    st.include(page: "sidepanel", it: app)
+    l.main_panel {
+        p {
+            raw("Click ")
+            a(href: "${app.rootUrl}${my.GDSL_URL}", target: "_blank") {
```

Comment:
:bug: use `rootURL`, not `app.rootUrl`. [prime directive](https://wiki.jenkins-ci.org/display/JENKINS/Hyperlinks+in+HTML)